include/fmt/compile.h

```diff
@@ -22,6 +22,10 @@
 #  endif
 #endif
 
+#ifndef FMT_ENABLE_FALLBACK_TO_RUNTIME_API
+#  define FMT_ENABLE_FALLBACK_TO_RUNTIME_API 1
+#endif
```

Comment:
I don't think we need this to be configurable considering that format string compilation is just an optimization. We don't provide any performance guarantees with or without compilation.

Reply:
Looks like I misunderstood you here - https://github.com/fmtlib/fmt/issues/2078#issuecomment-753644522:
> It can be sometimes useful to enforce the use of a specific API but it shouldn't be the default.